src/db/z_db.erl

```diff
@@ -748,7 +761,7 @@ create_schema(Site, Connection, Schema) ->
     end.
 
 %% @doc Check the information schema if a certain table exists in the context database.
--spec table_exists(string(), #context{}) -> boolean().
+-spec table_exists(table_name(), #context{}) -> boolean().
```

Comment:
The spec in line 764 uses a record, please define a type for the record and use that instead.